### PR TITLE
Added explanation about environment variables

### PR DESCRIPTION
**Pull request by nacho-undefinedlabs**
Added explanation about the environment variables, in line with Java docs, but adapted to Swift.

### CODE REVIEW

File: docs/swift-installation.md [hunk 1]
```diff
@@ -180,6 +181,57 @@ defaults write com.apple.dt.Xcode UseSanitizedBuildSystemEnvironment -bool NO
 
 After these steps, after running your tests locally in Xcode, you will get a test report in Scope with the results.
 
+#### SCOPE_DSN
```
Comment (fermayo) [inline docs/swift-installation.md]: Doesn't it make more sense to have this just before `Local development integration`?

File: docs/swift-installation.md [hunk 2]
```diff
@@ -180,6 +181,57 @@ defaults write com.apple.dt.Xcode UseSanitizedBuildSystemEnvironment -bool NO
...
+
+`SCOPE_DSN` contains the Data Source Name with connection information about the Scope installation to report to. Specifically, it contains the hostname of the Scope instance to report to, and the API key to be used for authentication.
+
+Scope DSNs are generated per namespace. You can generate them by clicking on the `API Key` link of the namespace where you want to report to in the Scope UI.
+
+#### SCOPE_XCODE_DSN
+
+This is a required setting when running tests locally.
+
+`SCOPE_XCODE_DSN`is used if `SCOPE_DSN` is not found.  If running locally using _Scope for Mac_ the `SCOPE_XCODE_DSN` environment variable is autogenerated when configuring the native application, and set when _Scope for Mac_ runs.
```
Comment (fermayo) [inline docs/swift-installation.md]:
```suggestion
`SCOPE_XCODE_DSN` is used if `SCOPE_DSN` is not found.  If running locally using _Scope for Mac_ the `SCOPE_XCODE_DSN` environment variable is autogenerated when configuring the native application, and set when _Scope for Mac_ runs.
```

File: docs/swift-installation.md [hunk 3]
```diff
@@ -180,6 +181,57 @@ defaults write com.apple.dt.Xcode UseSanitizedBuildSystemEnvironment -bool NO
...
+
+> Please, make sure  _Scope for Mac_ is running before opening the app that will run the tests
```
Comment (fermayo) [inline docs/swift-installation.md]:
```suggestion
> Please, make sure  _Scope for Mac_ is running and configured before opening the app that will run the tests
```

File: docs/swift-installation.md (hunk 1 above)
Comment (nacho-undefinedlabs) [reply docs/swift-installation.md]: Yes, moved CI env variables before Local development, and left SCOPE_XCODE_DSN after

**Pull request merged**